code/modules/crafting/craft.dm

```diff
@@ -106,6 +106,7 @@
 				var/list/parts = del_reqs(R, user)
 				var/atom/movable/I = new R.result (get_turf(user.loc))
 				I.CheckParts(parts, R)
+				user.put_in_hands(I)
```

Comment:
make sure it's actually an item before you put it in hands, or you'll get runtimes